frontend/src/Components/RoadDetails/TopBar.tsx

```diff
@@ -28,34 +27,19 @@ const return_btn = (
 /**
  * The Topbar with return and toggle button inside
  */
-const TopBar: React.FC<TopBarProps> = ({ isToggleOn }) => {
+const TopBar: React.FC<TopBarProps> = () => {
   const navigate = useNavigate(); // Get the navigate function
 
   const handleReturn = () => {
     // Navigate back to the home page when the button is clicked
     navigate('/');
   };
 
-  const handleToggleMiddleArea = () => {
-    // Toggle the state to show/hide MiddleArea2
-    isToggleOn((prevState) => !prevState);
-  };
-
   return (
     <div className="top-bar topBar-container ">
       <Link to="/" onClick={handleReturn} className="btnLinkContainer">
         {return_btn}
       </Link>
-      <div className="toggle-container">
-        <div className="toggle-label">
-          <span className="toggle-text">Map mode</span>
-          <label className="toggle-switch">
-            <input type="checkbox" onChange={handleToggleMiddleArea} />
-            <span className="slider"></span>
-          </label>
-          <span className="toggle-text">Road mode</span>
```

Comment:
> Could you remove the unused classes from the css files?

from PR 1

Reply:
but should we delete the other pull request and use only this one?

Reply:
Yes it's simpler and faster